### ISSUE TITLE
Projects page: Allow multiple filters to be “active” at one time(eg ‘tech’, ‘stages’, ‘events’, ‘topics’). Show those filters on the screen to the user.

### ISSUE THREAD

**Issue by titlecharacter** [CfABrigadePhiladelphia/laddr#98]
The projects page currently can only be viewed by one tag, stage, etc at a time. A member may, though, be looking for a combination of tags, or may be interested in more than one stage. 

For example: "I want to find javascript projects that are not drifting or hibernating."

**Comment by titlecharacter** [comment]
This would also set the stage for filtering projects out from default views that are "dead" in some respect (vaguely defined, never went anywhere, etc) but that we don't actually want to _delete._

**Comment by rmcmillen50** [comment]
https://codeforphilly.org/projects

**Comment by aharley40** [comment]
@rmcmillen50 How about adding a tiny plus on each filter, with a hover bubble saying 'Combine Filter'?

**Comment by rmcmillen50** [comment]
Yes, and...... Ideally we show something similar to say, LinkedIn or any job search site, where the user sees all the active filters in play. Let me see if I can attach this screenshot mockup I just made!

**Comment by rmcmillen50** [comment]
![image](https://cloud.githubusercontent.com/assets/7006568/23752165/307b156e-04a2-11e7-8f06-676cf950ac76.png)

**Comment by rmcmillen50** [comment]
so @aharley40, I added the "plus signs" you mentioned (well, that i think you meant :) ) on the left hand side. Then I suggest we add a horizontal section above the list of projects that shows what active filters are being used. You mentioned the "combine filters" idea.... I think we should just combine filters by default if I user clicks multiple. Then, the visual X's on each active filter would give people the ability to remove the filter if they wish. thoughts?

**Comment by aharley40** [comment]
Good Idea. I'll add a bubble to the top displaying the currently used tags. I could only include the deletion x's if we use the combine filter idea. Currently, clicking a filter causes a page refresh, and, I don't think the user will expect it to combine unless explicitly told. I think it'd be easier on the user to display help bubbles as they hover over the icons to add or delete the filter from the list.

**Comment by rmcmillen50** [comment]
well - to be clear - I think the most ideal experience is async project results updating when a user selects additional filters to their "query". So page doesn't refresh, just the project results section. The platform would be essentially assuming each additional tag is adding a new WHERE  (AND) clause condition to the SQL.
